test/test_ious.py

```diff
@@ -0,0 +1,147 @@
+from typing import List, Callable
+
+import pytest
+import torch
+import torch.fx
+from torch import Tensor
+from torchvision import ops
+
+
+class IouTestBase:
```

Comment:
Minor detail: `class IoUTestBase` would better match the `IoU` way of writing it. Not sure if this is the recommended way but I would write it that way if I had to choose.

Reply:
Also, it would be better to have the same convention as the other test classes (that starts with `Test`), i.e. something like:

`TestIoUBase`

Reply:
Yeah Let me Rename all and keep things consistent.

Reply:
Our current convention is to name the test function as Test<op_name>  so I have refactored accordingly for consistency. Nice catch @yassineAlouini